ehsm_kms_service/function.js

```diff
@@ -333,6 +342,26 @@ const _checkPayload = function (req, res) {
           return false
         }
       }
+      if (currentPayLoad[key].type == 'base64' && payload[key]) {

```

Comment:
why not "else if" in _checkPayload

Reply:
I just try to follow the code  style like "string", "int" and else other

I dont known why the checking code dont use "else if" before too - -///.